<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>roomManager</class>
 <widget class="QWidget" name="roomManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="addRoomBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>90</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>添加客房</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>30</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
    </font>
   </property>
   <property name="text">
    <string>客房管理</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delRoomBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>删除客房</string>
   </property>
  </widget>
  <widget class="QPushButton" name="updateRoomBtn">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>更新客房</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
